conan/tools/gnu/pkgconfigdeps.py

```diff
@@ -113,12 +170,19 @@ def content(self):
         pc_files = {}
         host_req = self._conanfile.dependencies.host
         for require, dep in host_req.items():
+            # Restart the aliases cache per dependency
+            self._pkg_config_name_aliases = dict()
```

Comment:
this sounds a bit fragile. Maybe it would be better if it is not a class attribute, but passed around as argument or returned from methods.

Reply:
Agree. I will come up with another approach.